API_url_tracker.Rmd

```diff
@@ -0,0 +1,60 @@
+---
+title: "API URL tracker"
+author: "Oluwasegun Apejoye"
+date: "2023-09-04"
+output: html_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+
+# install api branch from experience dashboard
+# remotes::install_github("CDU-data-science-team/experiencesdashboard@latest_api_interface")
+# library(experiencesdashboard)
+
+# temporarily import the required function from the experiencesdashboard package
+source("https://raw.githubusercontent.com/CDU-data-science-team/experiencesdashboard/latest_api_interface/R/fct_api_pred.R")
```

Comment:
I might be misunderstanding, but why not just load the functions from the package? This seems a bit brittle